erp/views.py

```diff
@@ -342,6 +342,7 @@ def get_context_data(self, **kwargs):
             )
             context["erp"] = erp
             if erp.has_accessibilite():
+                erp.accessibilite.cheminement_ext_presence = not erp.accessibilite.cheminement_ext_presence
```

Comment:
I'd rather do this in the form constructor, what do you think? Also we should definitely add a comment explaining why we're doing this.

Reply:
Hold on, we shouldn't mutate the model instance but rather render its opposite value at rendering time. We need to rethink this.